src/oop/app/src/Parsers/FilmixParserStrategy.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+namespace src\oop\app\src\Parsers;
+
+use src\oop\app\src\Models\Movie;
+use src\oop\app\src\Parsers\ParserInterface;
+
+class FilmixParserStrategy implements ParserInterface
+{
+    private Movie $movie;
+
+    public function __construct()
+    {
+        $this->movie = new Movie();
```

Comment:
please try to replace creating of new instance with dependency injection ( your __construct should recice Movie as param)

Reply:
done